src/concurrent/mod.rs

```diff
@@ -201,7 +506,22 @@ pub struct WccOutput {
     pub ignored_files: Vec<String>,
 }
 
-struct SpaceData {
+impl WccOutput {
+    const fn new(
+        files: Vec<FileMetrics>,
+        project: ProjectMetrics,
+        ignored_files: Vec<String>,
+    ) -> Self {
+        Self {
+            files,
+            project,
+            ignored_files,
+        }
+    }
+}
+
+#[derive(Clone, Copy)]
```

Comment:
Same as above, I derived `Copy` so that I don't have to use a reference when it is passed to other functions like `MetricsData::function(space_data, ...)`.